src/commands/deploy.ts

```diff
@@ -252,7 +252,8 @@ export async function main(): Promise<void> {
     debug('\n' + error.stack)
     failWithSpinner('Could not upload content', error)
   }
-  return
+
+  process.exit(0)
```

Comment:
0 means success, we should use the `process.exitCode` by default

```suggestion
  process.exit()
```

Reply:
Removed the 0. Yes, it is a successful exit.